app/Enums/icon-name.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum IconNameEnum{
+    Add = 'add',
+    AddPlayList = 'addplaylist',
```

Comment:
icon name can not be concrete component icon 

main design principle: code to interface not to concrete implementation